### DISCUSSION TITLE
.EPS image conversion to .PNG

### DISCUSSION THREAD

**Discussion by valentinacardile** [ImageMagick/ImageMagick#3343] [Digital Image Processing]
Hello,

I have an issue for the .Eps files conversion to PNG file: the resulting image have a corrupted color correction (seems darker)

This is the command that I use:

convert -colorspace rgb -units PixelsPerInch -antialias -background none -density 300 -gravity center -quiet -resize x1280 "source.eps"[0] -density 72 -profile sRGB-IEC61966-2.1.icc "target.png

Can you please kindly help me?
Best regards
Valentina

**Comment by fmw42** [comment]
You should be using -colorspace sRGB not RGB. RGB is linear and will be darker. sRGB is non-linear and will be lighter. You should also do the -resize after reading the input file. -gravity center is likely not doing anything.

**Comment by snibgo** [comment]
As fmw42 says.

Also note that the OP command assigns a sRGB profile to a linear RGB image. If you really do want a linear result, you should assign a linear RGB profile. But I suspect you really want a sRGB result.
